test/StringSource-test.js

```diff
@@ -234,13 +234,16 @@ describe("StringSource", function() {
             let source = new StringSource(AST);
             let result = source.toString();
             assert.equal(result, "This is Example！？");
+            // The second whitespace
             assert.deepEqual(source.originalPositionFromPosition({
                 line: 1,
                 column: 8
             }), {
                 line: 1,
-                column: 9
+                column: 8
```

Comment:
Because the position `(1, 8)` in the generated sentence represents the second whitespace, which is before a link expression, its position in the original sentence should also be `(1, 8)`.